packages/data-point/lib/reducer-entity/resolve.js

```diff
@@ -6,17 +6,17 @@ const BaseEntity = require('../entity-types/base-entity/resolve')
  * Resolve an Entity Reducer, actual entity resolution is delegated
  * to each Entity.resolve method
  * @param {Object} manager
- * @param {function} resolveReducer
+ * @param {Function} resolveTransform
  * @param {Accumulator} accumulator
  * @param {ReducerEntity} reducerEntity
  * @returns {Promise<Accumulator>}
  */
-function resolve (manager, resolveReducer, accumulator, reducerEntity) {
+function resolve (manager, resolveTransform, accumulator, reducerEntity) {
```

Comment:
shouldnt this have been renamed to resolveReducerList ? I really think we should get rid of the word 'transform' in this context